src_exe/COCOShpc.cpp

```diff
@@ -433,94 +434,27 @@ int main (int argc, char* argv[])
 {
 
     std::cout << "\n\nSTART COCOS" << std::endl;
-
-
-    //Set Parameters, all of them can be changed later on manually in the executable
-    long long startevaltime = 0;
-    long long endevaltime = 0;
-    bool timelimitation = false;   
-    
-    int taubeg = -250e3;
-    int tauend =  250e3;
-    int binnumber = 2000;
-    	
-
-
+    Settings vSetting;
+    vSetting.ReadSettingsFile();
+    vSetting.PrintSettings();
 
     //#############################################################################//
     //############# Jetzt kommen jede Menge manuelles-Einstellen-Zeug #############//
-    std::string calibrationmodestring[3];
-    calibrationmodestring[0] = "none";
-    calibrationmodestring[1] = "read";
-    calibrationmodestring[2] = "write";
 
     int calibrationmode = 0;
-    std::string calibfilename = "Calibration.txt";
-
-
-    //Get settings from settings file
```

Comment:
Das sollte passen, weil jetzt die settings.txt in der Klasse eingelesen wird. Aber schau mal nochmal drueber.